### PR TITLE
Two changes

### PR DESCRIPTION
**Pull request by smileqiao1988**
Hello, I made two changes. Please review. Thank you.

### CODE REVIEW

File: ch05/ex5_23.cpp [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
-int main(void)
-{
-	int a, b;
-	cin >> a >> b;
-	cout << static_cast<double>(a) / b << endl;
-	
-	return 0;
-}
\ No newline at end of file
+
```
Comment (pezy) [inline ch05/ex5_23.cpp]: IMHO, It is a little incomprehensible for this question... (Even though the answer is not my contribution...)

File: ch03/ex3_17.cpp [hunk 2]
```diff
@@ -30,7 +30,7 @@ int main()
 
   for (decltype(vec.size()) i=0; i != vec.size(); ++i)
   {
-    if (i%8 == 0) cout << endl;
+    if (i!=0&&i%8 == 0) cout << endl;
     cout << vec[i] << " ";
```
Comment (pezy) [inline ch03/ex3_17.cpp]: Agree, and I also fix it in 2c3c9b3b453706c680e1a0f3f1da0eb7d8eb306b

**Comment by Mooophy** [comment]
Thx

**Comment by smileqiao1988** [comment]
Thank you. :)

**Pull request merged**